### DISCUSSION TITLE
如何获取一个 PAGFile 的 Graphic 的大小呢？

### DISCUSSION THREAD

**Discussion by yufengvac** [Tencent/libpag#2346] [Q&A] [answered]
我们需要在 js  代码层面监控PAGFile的内存占用，目前想以 PAGViewer 面板的 “Graphic” 的数值作为标准，请问有没有接口可以直接获取到这个值呢？
<img width="291" alt="image" src="https://github.com/Tencent/libpag/assets/7565523/7e4f35e8-fac7-431c-ac62-b849d6ac4013">

尝试使用过 pagView.player.graphicsMemory() 但拿到的好像不准，似乎这个值跟 Canvas 尺寸有关，但我看 PAGViewer 里面的 graphic 的值都是固定的
<img width="915" alt="image" src="https://github.com/Tencent/libpag/assets/7565523/b00f6b85-f176-4e4c-bd15-ff09b2f801cf">

**Comment by kevingpqi123** [comment, marked as answer]
这里只能获取到自己申请的，获取不到完整的，只是作为参考使用的
